tests/utils.go

```diff
@@ -40,3 +49,55 @@ func RunOnNode(node string, command string) (string, error) {
 
 	return outStrippedString, err
 }
+
+func GenerateBridgeNameAndResource() coreapi.ResourceName {
```

Comment:
Pls rename, we don't generate the bridge name anymore
GenerateBridgeNameAndResource -> GenerateResourceName

Reply:
Done